tgui/packages/tgui/styles/interfaces/LobbyMenu.scss

```diff
@@ -0,0 +1,182 @@
+.theme-crtgreen {
+  .LobbyScreen {
+    &.filterEnabled {
+      box-shadow: 0 0 15px;
+    }
+
+    .typeEffect {
+      overflow: hidden;
+      white-space: nowrap;
+      width: 0;
+      animation: typing 1s steps(20, end) forwards;
+      animation-delay: 1.3s;
+    }
+
+    .buttonEffect {
+      overflow: hidden;
+      white-space: nowrap;
+      width: 0;
+      animation: typing 1s steps(20, end) forwards;
+    }
+
+    .loadEffect {
+      height: 0;
+      animation: loading 1s steps(20, end) forwards;
+      animation-delay: 1.3s;
+    }
+
+    .sectionLoad {
+      animation: section-loading 1s forwards;
+      box-shadow: 0 0 15px;
+      background-color: rgba(0, 17, 0, 0.7);
+    }
+
+    .dividerEffect {
+      animation: typing 1s steps(20, end) forwards;
+      width: 0;
+    }
+
+    .bgLoad {
+      opacity: 0;
+      animation: bg-load 10s forwards;
+      animation-delay: 5s;
+    }
+
+    @keyframes bg-load {
+      from {
+        opacity: 0;
+      }
+      to {
+        opacity: 1;
+      }
+    }
+
+    @keyframes typing {
+      from {
+        width: 0;
+      }
+      to {
+        width: 100%;
+      }
+    }
+
+    @keyframes loading {
+      from {
+        height: 0;
+      }
+      to {
+        height: 100%;
+      }
+    }
```

Comment:
Width and height animations are slow in CSS, so it could be a contributing factor to the laggy UI experience.

Reply:
Check https://developer.mozilla.org/en-US/docs/Learn_web_development/Extensions/Performance/CSS for more details